example/lib/screens/navigation/tree_view.dart

```diff
@@ -231,6 +231,7 @@ TreeView(
         TreeViewItem(
           content: const Text('Home Remodel'),
           value: 'home_remodel',
+          disabled: true,
```

Comment:
I think a new item can be created to showcase the disabled item functionality instead of using one that already exists.